src/components/Editor/Utilities/ColorParser.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div><span v-html="parsedText" /> <code v-if="showRaw">({{ text }})</code></div>
+</template>
+
+<script>
+export default {
+  props: ['text', 'showRaw'],
+  created() {
+    const randomize = this.randomize;
+    setInterval(() => {
+      Array.from(document.getElementsByClassName('mc-obfuscated')).forEach((element) => {
+        element.textContent = randomize(element.textContent);
+      });
+    }, 100);
+  },
+  methods: {
+    randomize(text) {
+      let randomisedText = '';
```

Comment:
Probably best to keep spelling consistent here, go for the US spelling of `randomized`